src/main/resources/db/migration/h2/V1_235_1__AddExternalTestSecurityGroups.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO darts.user_account (usr_id, user_name, user_email_address, account_guid, is_system_user) values (-40, 'Xhibit', 'xhibit@hmcts.net', '43cb3e21-b03a-4629-9908-b0ddaf7448b9', true);
```

Comment:
Do we need to update the `modernised_darts_standing_data.sql` with these new system user details?

Reply:
no as these are out test users and not ones for production.